bussiness-logic-module/src/main/java/edu/hillel/App.java

```diff
@@ -38,5 +44,42 @@ public static void main( String[] args )
 //        // Виклик методу біна
 //        System.out.println(itemService.getAllItems());
 
+
+//        UserRepository userRepository = UserRepositorySQLImpl.getSingletonInstance();

```

Comment:
same remark as for other PRs test functionality in test folder not in main method